.gitattributes

```diff
@@ -1 +1 @@
-bin/* filter=lfs diff=lfs merge=lfs -text
```

Comment:
What was the reason behind deleting this configuration?

Reply:
My understanding of this line is that if enables git large file system on the ./bin directory. This tripped us up at one point. 

As the bin dir is in the ignore file and lfs didn't appear to be needed we removed the line. 

Sorry if this is something that is needed